cli/run.go

```diff
@@ -56,7 +56,15 @@ var (
 	logPrefixSeparatorStyle = lipgloss.NewStyle().Foreground(grayColor)
 )
 
+var runExampleText string
+
 func runCommand() *cobra.Command {
+	if isDocsBuild {
+		runExampleText = "```\ndispatch run [options] -- <command>\n```"
+	} else {
+		runExampleText = "  dispatch run [options] -- <command>"
+	}
```

Comment:
Same idea here, `runExampleText` could be a constant defined with a different value in each of the `generate_docs*` files.